README.md

```diff
@@ -1028,6 +1027,31 @@ In larger expressions, optional parentheses can sometimes make code read more cl
 let playerMark = (player == current ? "X" : "O")
 ```
 
+## Fatal errors
+
+Avoid fatal errors if possible.
+
+**Preferred:**
+
+```swift
+guard views.count > 0 else {
+    fatalError("Views must contain at least one view")
+  	return
+}
+```
+
+**Not Preferred:**
+
+```swift
+guard views.count > 0 else {
+  	return
+}
+```
```

Comment:
Seems like these cases should be swapped 🤔

Reply:
I'd expand on this topic a bit, e.g.:
- use assertions or exceptions instead where it makes sense
- cases where using a fatalError would be acceptable (as we discussed in our meeting)

Reply:
Could you help me with cases and examples?